src/controllers/species.controller.ts

```diff
@@ -86,6 +88,25 @@ export class SpeciesController {
     await this.speciesRepository.updateById(id, species);
   }
 
+  @post('/species/combine', {
+    responses: {
+      '204': {
+        description: 'Species POST success',
+      },
+    },
+  })
+  async combine(
+    @requestBody() request: { combine: number[]; species: Species },
+  ): Promise<void> {
+    const newSpecies = await this.speciesRepository.create(request.species);
+    console.log(newSpecies);
+    for (const id of request.combine) {
```

Comment:
Seems with Loopback, is easy to wrap the executions in a transaction, for this `for` loop, it might bring some safety. 
Or, could we think about use bulk update to reduce the connection count to the DB? Like: `update set a=x where id in (1,2,3)`

Reply:
That sounds interesting! I will try to implement the bulk update version. Thanks for the feedback.

For the testing, I was using it in parallel with the client to manually add/combine species and also using Postman. Do we have any other way to test it?